managed/services/management/service.go

```diff
@@ -56,6 +56,25 @@ func NewServiceService(db *reform.DB, state agentsStateUpdater, vmdb prometheusS
 
 // RemoveService removes Service with Agents.
 func (s *ServiceService) RemoveService(ctx context.Context, req *managementpb.RemoveServiceRequest) (*managementpb.RemoveServiceResponse, error) {
+	if e := s.db.InTransaction(func(tx *reform.TX) error {
+
+		agent, err := models.FindAgentByID(tx.Querier, req.AgentId)
+		if err != nil {
+			return err
+		}
+		serviceType := serviceTypes[req.ServiceType]
```

Comment:
Do not pay attention to this line. I'll make it panic-free soon. It's just for PoC demo